tests/test.hs

```diff
@@ -158,7 +158,7 @@ knownToFail Z3   = [ "tests/pos/linspace.hs" ]
 testCmd :: FilePath -> FilePath -> FilePath -> SmtSolver -> LiquidOpts -> String
 ---------------------------------------------------------------------------
 testCmd bin dir file smt (LO opts)
-  = printf "cd %s && %s --verbose --smtsolver %s %s %s" dir bin (show smt) file opts
+  = printf "cd %s && %s --smtsolver %s %s %s" dir bin (show smt) file opts
```

Comment:
Plus:

+ Don't want to have the numbers generated by `stack/cabal test` to be corrupted by debugging, as they are also used to measure performance over time,

+ We can always add the `--verbose` as an extra parameter to `stack/cabal test` if we really want to generate the verbose logs on travis, no need to hardwire the thing inside the script...